tests/qa_automation/execute_test_run.pm

```diff
@@ -19,6 +19,7 @@ use testapi;
 use ctcs2_to_junit;
 use upload_system_log;
 use base "opensusebasetest";
+use version_utils qw(is_jeos);
```

Comment:
Please, use single quotes in case of single import.

Reply:
Updated.
Here is a new validation run: http://ccret.suse.cz/tests/2397#

Reply:
You actually don't need verification run for this kind of changes, I trust that perl import work fine =)